pkgs/by-name/ga/gama-tui/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "gama-tui";
+  version = "1.1.4-unstable-2024-09-05";
+
+  src = fetchFromGitHub {
+    owner = "termkit";
+    repo = "gama";
+    rev = "5063d35e48f3118fa793965fcbb931ef3b330f02";
```

Comment:
Why is the release version not enough?

Reply:
I honestly don't remember, I was using unstable version for a while locally.

changed to tagged release